.github/workflows/docs.yml

```diff
@@ -0,0 +1,58 @@
+name: Build Documentation
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: "The version string for this release"
+        required: true
+        type: string
+      pdf-name:
+        description: "The name of the PDF file generated by latexpdf, as set in your conf.py"
+        required: true
+        type: string
+      python-version:
+        description: "The version of Python used to build the docs, defaults to 3.8.10"
+        default: 3.8.10
+        required: false
```

Comment:
this is required even if it has a default value. If someone remove the default it will run anyway.

Reply:
After discussion: it's better to pin dependencies to avoid surprises and inject unknown packages.